src/main/java/at/hannibal2/skyhanni/features/fishing/tracker/SeaCreatureTracker.kt

```diff
@@ -151,10 +157,20 @@ object SeaCreatureTracker {
         tracker.renderDisplay(config.position)
     }
 
+    @SubscribeEvent
+    fun onRepoReload(event: RepositoryReloadEvent) {
+        val data = event.getConstant<ItemsJson>("Items")
+        trophyArmorNames = data.trophy_fishing_armors.toSet()
+    }
+
     fun resetCommand(args: Array<String>) {
         tracker.resetCommand(args, "shresetseacreaturetracker")
     }
 
     private fun isEnabled() = LorenzUtils.inSkyBlock &&
-        FishingAPI.lastActiveFishingTime.passedSince() < 10.minutes && config.enabled
+        FishingAPI.lastActiveFishingTime.passedSince() < 10.minutes && config.enabled && !isTrophyFishing()
+
+    private fun isTrophyFishing(): Boolean {
+        return trophyArmorNames.containsAll(InventoryUtils.getArmor().map { it?.getInternalName() })
```

Comment:
Isnt simply one regex of `(bronze|silver|gold|diamond)_hunter_(helmet|chestplate|leggings|boots)` better?

Reply:
I like this much better and this can also be in the repo through a repo pattern

Reply:
Even simpler would be to just check the lore? 
![image](https://github.com/hannibal002/SkyHanni/assets/69345714/70a921be-930d-4653-a3c1-f09357035322)
If it says Peace Treaty (4/4)

Reply:
this value bugs out some times so I wouldn't use it. Also lore checking is expensive

Reply:
i don't think a pattern is super needed since hypixel doesn't usually change item ids, but ill do this